tests/framework/ensembleModelTests/checkReusability/alpha.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
```

Comment:
As you mentioned, these files are reused in some ensemble model tests, so it might be convenient to move them to a common folder. Not required (by me at least).

Reply:
Done